examples/thermostat/efr32/src/SensorManager.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**********************************************************
+ * Includes
+ *********************************************************/
+
+#include "SensorManager.h"
+#include "AppConfig.h"
+#include "AppEvent.h"
+#include "AppTask.h"
+
+#ifdef USE_TEMP_SENSOR
+#include "TemperatureSensor.h"
+#endif
+/**********************************************************
+ * Defines and Constants
+ *********************************************************/
+
+#define ENDPOINT_ID 1
```

Comment:
nit: `constexpr EndpointId kEndpointId = 1` might be more typesafe. Maybe consider all `#defines` to be C++ constexpr for type safety as well.

Reply:
done and applied to other defines